wiki/Glossary/en.md

```diff
@@ -528,29 +518,29 @@ This is to offset the beatmap's offset alongside the global offset.
 
 ### M4M
 
-M4M, or "Mod for Mod", is a term describing when a modder agrees to mod another modder's map in return for a mod on their own map.
+M4M, or Mod for Mod, is a term describing when a modder agrees to mod another modder's map in return for a mod on their own map.
 
 ### Marathon
 
 A marathon is a type of beatmap that can only be approved.
 
 Marathon beatmaps are considered to be at least 6 minutes long and can only have one mapped difficulty.
-However, some marathon mapsets may contain an osu!taiko variant as well.
+However, some marathon mapsets may contain a single osu!taiko variant as well.
 
-Marathon beatmaps are sometimes collaborated together due to the length of the music.
+Some marathon beatmaps are sometimes collaborated due to the length of the music.
```

Comment:
"Marathon beatmaps are often collaborative efforts, as mapping longer tracks is a draining endeavour when done solo."